integration-test/2095_inline_layers.py

```diff
@@ -0,0 +1,78 @@
+# -*- encoding: utf-8 -*-
```

Comment:
did you check whether your PR introduces new test failures than the master branch?

Reply:
do we have a list of ones that are failing? i can check on my machine which ones fail and pass before and after my change and report back

Reply:
you can also check it on the CI report

Reply:
the report is identical to masters last failed build, the tests fail at: `test.test_transform.BuildingsUnifyTest`

Reply:
oh no, this is not good, it means none of integration tests got a chance to even run yet. I found the unit test failure was introduced recently https://github.com/tilezen/vector-datasource/pull/2093

@travisgrigsby would you help fix that unit test so that other integration tests can be run and checked?

Reply:
done #2099

Reply:
cool ill wait for that to merge, then merge master into this branch then compare the two outputs from CI

Reply:
i was finally able to get the build to run al the way and it has the same number of failures as master for integration tests:

master:
```
----------------------------------------------------------------------
Ran 2198 tests in 1000.156s

FAILED (failures=109)
```

this branch:
```
----------------------------------------------------------------------
Ran 2200 tests in 734.782s

FAILED (failures=109)
```